### PR TITLE
Instructional Design Documentation

### PR DESCRIPTION
**Pull request by jswope00**
Adding discussed documentation for instructional Design:

Instructional Design Overview
Course Development Process
Instructional Design Templates

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jswope00!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mphilbrick211** [comment]
Hi @jswope00! Just checking to see if this is ready for review?

**Comment by jswope00** [comment]
Hi @mphilbrick211 yes, this is ready to be reviewed and merged. Thanks!

**Comment by mphilbrick211** [comment]
Hi @feanil! Could you take a look at this?

**Comment by openedx-webhooks** [comment]
@jswope00 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/docs.openedx.org&entry.867055334=https://github.com/openedx/docs.openedx.org/pull/233&entry.1484655318=https://github.com/jswope00&entry.752974735=2022-12-21+14:47&entry.1917517419=2023-01-23+20:52&entry.2133058324=No) so we can improve your experience in the future.

**Comment by mphilbrick211** [comment]
Sorry @feanil and @jswope00 - I closed this by accident - reopened for Feanil's review/merge.

**Comment by feanil** [comment]
👍🏾 I'll review it in the next week or so hopefully.  Also adding @jmakowski1123 as a reviewer.

File: source/educators/concepts/instructional_design/id_templates.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+   :alt: Thumbnail of Downloadable Course Outline Template
+
+.. _GradingChartTemplate:
+
+Grading Chart
+****************************************
+
+Most courses will have a variety of assessments that provide learners multiple opportunities to show evidence of their learning. A grading chart like the one below will help you track the assessments in the course and ensure that you have correctly weighted all of the assessments. This chart can also help you to keep track of assignment deadlines for both learners and course instructors.
+
+`Download Grading Chart <https://docs.google.com/spreadsheets/d/1LjWAM5N5nFLMTNdFCDjq23ffJTDG-BH81jNjHrWusj0/edit?usp=sharing>`_
```
Comment (feanil) [inline source/educators/concepts/instructional_design/id_templates.rst:67]:
```suggestion
`Download Grading Chart <https://docs.google.com/spreadsheets/d/1wq3tlIXNqZyy0bL2Dn8hgO9UY0xQMwGbhct_MQ9DSZc/edit#gid=1193955468>`_
```

File: source/educators/concepts/instructional_design/id_templates.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+   :alt: Thumbnail of Downloadable Template for Grading Chart
+
+.. _QualityControlTemplate:
+
+Quality Control Template
+****************************************
+
+In the final stages of the course development process you will need to spend time reviewing your course for elements that include not only the course content and assessments, but the course about page, certificates (if applicable), and other items that can easily be forgotten during course development. This template should be edited to fit your organization or client’s needs, but it provides a template for many of the items that you should be checking before you launch your course. Keep in mind that reviewing for accessibility (image alt tags, transcripts, etc.) is an important part of this process.
+
+`Download Quality Control Template <https://docs.google.com/document/d/1ihzP51X26sU7yONFcRt8LBjvG1iXGM51VX9GT234fbQ/edit?usp=sharing>`_
```
Comment (feanil) [inline source/educators/concepts/instructional_design/id_templates.rst:79]:
```suggestion
`Download Quality Control Template <https://docs.google.com/document/d/1zWYVCI_M39-bHjaqoerGgVuD4_ZursvabzJSNLQBs9s/edit?usp=share_link>`_
```

File: source/educators/concepts/instructional_design/id_templates.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+   :alt: Thumbnail of Downloadable Quality Control Template
+
+.. _CommunicationPlanTemplate:
+
+Communication Plan Template
+****************************************
+
+One way to ensure learner engagement is to consistently communicate with your learners. There are various options for communicating with your learners and it is important to plan out through which methods and how often you will communicate with the learners in your course. This example communication plan provides some suggestions on items that should be included in your communication plan but will likely need to be modified to fit the needs of each organization.
+
+`Download Sample Communication Plan <https://docs.google.com/document/d/1b043Z85tq4oJQLmrADmiVOHdkMn4F9on/edit?usp=sharing&ouid=102215557727623303211&rtpof=true&sd=true>`_
```
Comment (feanil) [inline source/educators/concepts/instructional_design/id_templates.rst:91]:
```suggestion
`Download Sample Communication Plan <https://docs.google.com/document/d/1-ooDKyWrLvj6Z2pRud7douMEtPbHc69K/edit?usp=share_link&ouid=106462470317905634700&rtpof=true&sd=true>`_
```

File: source/educators/concepts/instructional_design/id_templates.rst [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+   :alt: Thumbnail of Downloadable Template for Communication Plan
+
+.. _SupportCalendarTemplate:
+
+Support Calendar Template
+****************************************
+
+If your team is involved in the delivery of a course, you may want to spend some time creating a support calendar (or encouraging your SMEs/instructors to create one). A support calendar lists all of the responsibilities that are needed for delivery from releasing content to providing assignment feedback and communicating with students. You can set deadlines and assign each item to a member of your team.
+
+`Download Support Calendar <https://docs.google.com/spreadsheets/d/1LjWAM5N5nFLMTNdFCDjq23ffJTDG-BH81jNjHrWusj0/edit?usp=sharing>`_
```
Comment (feanil) [inline source/educators/concepts/instructional_design/id_templates.rst:103]:
```suggestion
`Download Support Calendar <https://docs.google.com/spreadsheets/d/1wq3tlIXNqZyy0bL2Dn8hgO9UY0xQMwGbhct_MQ9DSZc/edit#gid=2000240391>`_
```

**Review by feanil — CHANGES_REQUESTED** [review]
This is really awesome, for the most part this looks ready to land to me, just two small things that need to be updated before it's ready to go.

* source/_images/instructional_design/addie_model.png can you update this to remove the curricu.me logo?
* I've updated the URLs for the templates so that they're hosted by tCRIL, can you accept those changes?

**Comment by jswope00** [comment]
@feanil and @jmakowski1123 changes made!

**Review by feanil — APPROVED** [review]
Looks great, thanks @jswope00!

**Review by jmakowski1123 — APPROVED** [review]
@feanil I checked all the template links and I'm able to edit the word docs. Is that because I have access to the drive, or are they not set to read only? Other than that, looks good!

**Comment by feanil** [comment]
@jmakowski1123 that is because you have access to the drive, I've confirmed that anonymous users only have read access.

**Comment by openedx-webhooks** [comment]
@jswope00 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/docs.openedx.org&entry.867055334=https://github.com/openedx/docs.openedx.org/pull/233&entry.1484655318=https://github.com/jswope00&entry.752974735=2022-12-21+14:47&entry.1917517419=2023-02-06+17:21&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**